eclair-core/src/test/scala/fr/acinq/eclair/testutils/MySlf4jLogger.scala

```diff
@@ -0,0 +1,142 @@
+package fr.acinq.eclair.testutils
+
+import akka.actor.{Actor, ExtendedActorSystem}
+import akka.dispatch.RequiresMessageQueue
+import akka.event.Logging.{Debug, Error, Info, InitializeLogger, LogEvent, LogEventWithCause, LogEventWithMarker, LoggerInitialized, Warning}
+import akka.event.slf4j.{SLF4JLogging, Slf4jLogMarker}
+import akka.event.{DummyClassForStringSources, LoggerMessageQueueSemantics}
+import akka.util.Helpers
+import ch.qos.logback.classic.LoggerContext
+import org.slf4j.{ILoggerFactory, MDC, Marker, MarkerFactory}
+
+/**
+ * Copy of [[akka.event.slf4j.Slf4jLogger]], with a customizable [[ILoggerFactory]] so we can separate logs in tests.
+ */
+
+object MyLogger {
```

Comment:
It's tempting to refactor this object, the code is quite messy and doesn't follow our codebase's style, but I understand the will to keep it close to the akka version.

Reply:
I'd like to try submitting a PR to include this in Akka but it's a long shot.